server/utils/index.js

```diff
@@ -4,7 +4,7 @@ const generateUsername = name => {
   let specials = /[`~!@#$%^&*()_|+\-=?;:'",.<>\{\}\[\]\\\/\s]/gi;
   let sanitized = name.toLowerCase().replace(specials, "");
   let uniqueId = uniqid.time();
-  return `${sanitized}_${uniqueId}`;
+  return `${sanitized}-${uniqueId}`;
```

Comment:
changed username strings to use dash instead of underscore mainly because currently there is a bug in Showdownjs where it does not detect the _usernames. showdownjs/showdown#803

But i think i liked the dash pattern better.